en-US/firefox/whatsnew_71.lang

```diff
@@ -0,0 +1,54 @@
+## NOTE: Demo https://www-demo1.allizom.org/firefox/71.0/whatsnew/all/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/71.0/whatsnew/all/
+
+
+# HTML page title.
+;What’s new with Firefox - More privacy, more protections.
+What’s new with Firefox - More privacy, more protections.
+
+
+# Notification banner.
+;Congrats! You’re using the latest version of Firefox.
+Congrats! You’re using the latest version of Firefox.
+
+
+# Page headline.
+;No account required. But you might want one.
+No account required. But you might want one.
+
+
+;The Firefox browser collects so little data about you, we don’t even require your email address. But when you use it to create a Firefox account, we can protect your privacy across more of your online life.
+The Firefox browser collects so little data about you, we don’t even require your email address. But when you use it to create a Firefox account, we can protect your privacy across more of your online life.
+
+
+;Sign In
+Sign In
+
+
+;Have at least one company looking out for your data, instead of leaking it.
+Have at least one company looking out for your data, instead of leaking it.
+
+
+;Never forget, reset or travel without your passwords again.
+Never forget, reset or travel without your passwords again.
+
+
+# The word container relates to a 'container tab', used by the Facebook Container extension.
+;Get a container to keep Facebook out of your business.
+Get a container to keep Facebook out of your business.
+
+
+;Trade clickbait and fake news for quality content.
+Trade clickbait and fake news for quality content.
+
+
+;Send huge files to anyone you want, with self-destructing links.
+Send huge files to anyone you want, with self-destructing links.
+
+
+;Get a Firefox Account
+Get a Firefox Account
+
+
+;Read the <a href="%(notes)s">Release Notes</a> to know more about what’s new in your Firefox Browser.
```

Comment:
Let's add a comment here on the capital B in Firefox Browser. Browser should be preferably left in English but can be translated due to linguistic necessity.